Fix XenAPI.py errors
    
    This commit fixes two bugs in XenAPI.py. One was probably a Python 2.4 -> 2.7
    bug, where we weren't initialising UDSTransport's superclass. Then next was a
    bug we introduced when we moved the XCP files around in /var: UDSTransport was
    relying on the socket /var/xapi/xapi, but we had moved it to /var/xcp/xapi.
    
    Signed-off-by: Mike McClurg <mike.mcclurg@citrix.com>

--- a/scripts/examples/python/XenAPI.py
+++ b/scripts/examples/python/XenAPI.py
@@ -96,6 +96,7 @@
 
 class UDSTransport(xmlrpclib.Transport):
     def __init__(self, use_datetime=0):
+        xmlrpclib.Transport.__init__(self, use_datetime)
         self._use_datetime = use_datetime
         self._extra_headers=[]
     def add_extra_header(self, key, value):
